src/hm-admin/hm-admin.component.ts

```diff
@@ -11,16 +11,16 @@ import {CoreDefinitionService} from './services/core/CoreDefinition.service';
   providers: [ConfigService, CoreDefinitionService, HTTP_PROVIDERS]
 })
 export class HmAdminComponent implements OnInit {
-  @Input('config')
```

Comment:
by default it keep the same name. It useful when you want to define an input with several words
in attribute, we considering it's a good habit to use kebab case and for javascript var to use camelCase.
so 
```
<hm-admin configuration-file="">


@Input('configuration-file')
configurationFile: any
```

But if the name is the same, you don't have to fill this parameter